src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.bar {
+  position: relative;
+  margin-top: 20px;
+}
+
+body {
+  font-family: Avenir, Helvetica, sans-serif;
+}
+
+.firstBarInput {
```

Comment:
Try to follow DRY principe (Don't repeat yourself). So here u can collect such rules as  border, box-shadow, border-radius, etc
And use them like this 
```
.firstBarInput,
.secondBarInput {
....your common rules
}
```